src/track/track.cpp

```diff
@@ -281,10 +281,10 @@ double Track::setBpm(double bpmValue) {
     if (!m_pBeats) {
         // No beat grid available -> create and initialize
         double cue = getCuePoint().getPosition();
```

Comment:
```suggestion
        mixxx::FramePos cue(getCuePoint().getPosition() / mixxx::kEngineChannelCount);
```
Let's minimize use of `double` variables.